reo/tests/test_offgrid.py

```diff
@@ -71,13 +71,13 @@ def setUp(self):
                     "annual_kwh": 87600.0,
                     "year": 2017,
                     "min_load_met_pct": 1.0,
-                    "sr_required_pct": 0.1
+                    # "sr_required_pct": 0.1

```

Comment:
@adfarth should this line and line 80 in the same file be commented out? (If so can you please add a comment to  test_offgrid explaining why.)

Reply:
@NLaws It won't impact the test outcomes (no hard-coded outputs are currently tested) but I think it may be better to have the default spinning reserve values used. I removed those lines in https://github.com/NREL/REopt_API/pull/296/commits/27d8cf31f5e75f8e07524598d2f89462989d5c4f. Does that sound ok?